### DISCUSSION TITLE
Release with four digits Feature.Major.Minor.Patch

### DISCUSSION THREAD

**Discussion by Jeenusck** [GitTools/GitVersion#3000] [Q&A] [answered]
Our products is using four digit versioning.  Is it possible to generate four digit version using gitversion.yml?

**Comment by asbjornu** [comment, marked as answer]
How do you want to consume the version number? Since GitVersion exposes all of its [version variables](https://gitversion.net/docs/reference/variables), you can build any version number you want in the build pipeline after GitVersion has executed. You can also adjust the [`assembly-versioning-scheme`](https://gitversion.net/docs/reference/configuration#assembly-versioning-scheme) to something like `'{env:FEATURE}.{Major}.{Minor}.{Patch}'` and consume that directly.

**Reply by Jeenusck** [reply]
We have release branch as well as hotfixes. For now feature and major version won't change. Only minor and patch versions will change.

We are using github actions. 
We think {env:FEATURE}.{Major}.{Minor}.{Patch} using gitflow will work for us.  We tried to CommitsSinceVersionSource but we need to support patch releases. So we cannot use CommitsSinceVersionSource.
